websites/Z/ZenMarket/presence.ts

```diff
@@ -0,0 +1,304 @@
+const presence = new Presence({
+		clientId: "971483473024004157",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.ibb.co/n3h8QbV/zenlogo-sm.png",
+}
+
+// Function to remove the lang prefixes that occur with different languages
+async function removeLangPath(path: string) {
+	const splitPath = path.split("/"),
+		locales: Set<string> = new Set([
+			"en",
+			"ru",
+			"de",
+			"ua",
+			"ja",
+			"cn",
+			"tw",
+			"es",
+			"fr",
+			"ms",
+			"vi",
+			"ar",
+			"id",
+			"th",
+			"it",
+			"pt",
+			"tr",
+			"pl",
+			"ko",
+		]);
+
+	if (locales.has(splitPath[1]))
+		return splitPath.filter(part => !locales.has(part)).join("/");
+	else return path;
+}
```

Comment:
Why not just do this?
```ts
location.pathname.replace(`/${document.querySelector('html').getAttribute('lang')}`, "")
```

![image](https://github.com/PreMiD/Presences/assets/42322979/fa4899d5-01d2-49d3-a083-80a779dd92f0)